infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -139,7 +142,6 @@ public ServerContext(Map<String, Object> serverConfig) {
         this.dataStore = new DataStore(serverConfig, this::dataStoreFileCleanup);
         generateNodeId();
         this.failureHandlerPolicy = new ConservativeFailureHandlerPolicy();
```

Comment:
Do we want to remove this from here and move it to the ManagementServer as we discussed?